microarray/resources/schemas/dbscripts/postgresql/microarray-19.20-19.21.sql

```diff
@@ -1,22 +1,18 @@
-/*

```

Comment:
Adam and I noticed this as well... I'm hoping this is just git/github being dumb but would be good for Rosaline to verify

Reply:
Not sure how FileStatus.java is being pulled in here, since I simply deleted it...
I remember creating the upgrade script by copying the previous upgrade script file contents and changing the  few lines necessary.